waltz-tools/src/main/java/com/wepay/waltz/tools/cluster/ClusterCli.java

```diff
@@ -613,13 +632,18 @@ private boolean verifyValidation(ValidationResult.ValidationType type,
 
             ValidationResult validationResult = partitionResult.validationResultsMap.get(type);
             if (validationResult.status.equals(ValidationResult.Status.FAILURE)) {
-                System.out.println("Validation " + type.name() + " failed for partition " + partitionId);
-                System.out.println("Validation error is: " + validationResult.error);
+                appendLineSB("Validation " + type.name() + " failed for partition " + partitionId);
+                appendLineSB("Validation error is: " + validationResult.error);
                 return false;
             }
             return true;
         }
 
+        private void appendLineSB(String toOutput) {
```

Comment:
May be better to move this to "CliUtils.java" instead of duplicating the same in all Cli files?